lib/webpack.js

```diff
@@ -132,3 +132,25 @@ exportPlugins(exports.node = {}, {
 exportPlugins(exports.debug = {}, {
 	"ProfilingPlugin": () => require("./debug/ProfilingPlugin"),
 });
+
+const defineMissingPluginError = (pluginName, errorMessage) => {
+	Reflect.defineProperty(exports.optimize, pluginName, {
+		configurable: false,
+		enumerable: true,
+		get() {
+			throw new Error(`RemovedPluginError: ${errorMessage}`);
```

Comment:
Make it a Error class: RemovedPluginError

Reply:
Should I write that inline here, or put it in a separate file somewhere?